plugins/com.google.cloud.tools.eclipse.appengine.login.test/src/com/google/cloud/tools/eclipse/appengine/login/ui/LoginServiceUiTest.java

```diff
@@ -96,10 +96,11 @@ private Thread createListenerThread(final ServerSocket serverSocket) {
     return new Thread(new Runnable() {
       @Override
       public void run() {
+        Charset utf8 = StandardCharsets.UTF_8;
```

Comment:
Does assigning this to a local variable really buy us anything?  `StandardCharsets.UTF_8` is a static field and it will point to the same instance.